tool/speed.cc

```diff
@@ -2112,9 +2112,7 @@ bool Speed(const std::vector<std::string> &args) {
      !SpeedScrypt(selected) ||
 #endif
      !SpeedRSA(selected) ||
-     !SpeedRSAKeyGen(selected)
```

Comment:
There's a duplicate on L2134, so I'm removing this one.

Reply:
If this was working I'd vote remove the one on 2147 as that's inside an `if !defined(OPENSSL_BENCHMARK)` block but clearly this benchmark does work with OpenSSL.

Reply:
I left both and modified the second one to use `RSA_generate_key_fips` (not supported by OpenSSL), and the first one to use `RSA_generate_key_ex`(supported by OpenSSL).

Reply:
Do/will we run both for EC_keygen as well?

Reply:
I briefly looked into it and the `SpeedECKeyGenCurve` uses `EVP_PKEY_keygen`, so it might require a larger refactor to call `EC_KEY_generate_key/EC_KEY_generate_key_fips`. I'll open a separate PR for that, so this one is focused on the RSA PWCT needed for FIPS.